### PR TITLE
split up css classes

### PR DESCRIPTION
**Pull request by FinnStutzenstein**
What do you think of this new way of managing all the css classes?

There will be two new gulp tasks that create two css files for either site and projector. The files included are given in the core/static/css/site.css (resp. projector.css) with import statements. The gulp task parses them and put everything together. Every app should get a site.css and  projector.css as well as css files for parts like the diff-view. These files gets imported by every app's site.css/projector.css.

This should be the the import order:
```
core/static/css/site.css
  - core/static/css/core/site.css
      - fonts.css
      - helper.css
      - mediaqueries.css
      - osTable.css
      - ...
  - motions/static/css/motions/site.css
      - diffView.css
  - users/static/css/users/site.css
  - ...
core/static/css/projector.css
  - core/static/css/core/projector.css
      - fonts.css
      - helper.css
      - slides.css
      - ...
  - motions/static/css/motions/projector.css
      - diffView.css
  - users/static/css/users/projector.css
  - ...
```

### CODE REVIEW

**Comment by normanjaeckel** [comment]
Please check #3253, #2209 and #3011. Of course we can change the way we handle CSS and we do not have to use the jspm loader but concat all together. I only recommend to think about this for a moment.

File: gulpfile.js [hunk 1]
```diff
@@ -102,6 +103,15 @@ gulp.task('templates', function () {
         .pipe(gulp.dest(path.join(output_directory, 'js')));
 });
 
+gulp.task('css_site', function () {
```
Comment (normanjaeckel) [inline gulpfile.js]: Please add docstring with short explanation.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel I switched to sass. Is this now ok? I'll do a _variables.scss with all things we use often (OS colors, alert boxes colors, sizes, fonts, ..).

**Comment by normanjaeckel** [comment]
Ok, looks good. I do not understand why we use and underscore here but if you do it is ok for me.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel I do not like the underscore neither, but this is [sass syntax for partial stylesheets](http://sass-lang.com/documentation/file.SASS_REFERENCE.html#Partials__partials):
`If you have a SCSS or Sass file that you want to import but don't want to compile to a CSS file, you can add an underscore to the beginning of the filename. This will tell Sass not to compile it to a normal CSS file. You can then import these files without using the underscore.`

**Comment by FinnStutzenstein** [comment]
You can test the site. should be ok for the first splitting.

**Comment by emanuelschuetze** [comment]
@normanjaeckel Ready for next review.

**Comment by FinnStutzenstein** [comment]
Sass itself does neither globs nor variables in imports. This package enables glob support: https://www.npmjs.com/package/gulp-sass-glob

I'll try this

**Comment by emanuelschuetze** [comment]
@FinnStutzenstein Please ping if this pr is ready to merge.

**Comment by FinnStutzenstein** [comment]
@normanjaeckel I tried to use sass-glob, but somehow it does not work and I'm not exactly sure why. So lets lust merge this PR.

**Pull request merged**